Tools/config/config.json.sample

```diff
@@ -1,5 +1,7 @@
 {
+  "service-url": "https://veupathdb.org/dataset-import/user-datasets"
+  "super-user-token": "sfischer%40upenn.edu-a-very-long-number"
```

Comment:
Do we have a plan for where to ultimately fetch this from?

Reply:
the token will be for the super user account.  we can scrape it from a website request